lib/ex_doc/formatter/html.ex

```diff
@@ -87,6 +91,21 @@ defmodule ExDoc.Formatter.HTML do
     write_readme(output, File.read(readme_path), module_nodes, config, modules, exceptions, protocols)
   end
 
+  defp process_logo_metadata(config) do
+    logo_path = Path.expand config.logo
+    output = "#{config.output}/assets"
+    File.mkdir_p! output
+    file_extname = Path.extname(config.logo) |> String.downcase
+
+    if file_extname == ".png" || file_extname == ".jpg" do
```

Comment:
Wow, another thing that I didn't know :smiley: 

Where I can find more info about this?

BTW, thanks for the detailed review :smile: